README.md

```diff
@@ -2,16 +2,18 @@
 [![npm version](https://badge.fury.io/js/generator-symphony.svg)](https://badge.fury.io/js/generator-symphony)
 [![CircleCI](https://circleci.com/gh/SymphonyPlatformSolutions/generator-symphony.svg?style=shield)](https://circleci.com/gh/SymphonyPlatformSolutions/generator-symphony)
 
-Yeoman based generator for Symphony Bots and Applications in 2 different languages:
-- [Java](https://github.com/finos/symphony-bdk-java)
-- [Python](https://github.com/finos/symphony-bdk-python)
+Yeoman based generator for Symphony Bots and Applications:
+- [Java bot](https://github.com/finos/symphony-bdk-java)
+- [Python bot](https://github.com/finos/symphony-bdk-python)
+- [WDK (Symphony Worklfow Developer Kit)](https://github.com/SymphonyPlatformSolutions/symphony-wdk)
 
 ## Quick Start
 
 1. Install Yeoman command line utility `npm install -g yo`
 2. Install Symphony BDK Generator `npm i -g generator-symphony`
-3. Create a new directory and go into it `mkdir myBot && cd myBot`
-4. Run Symphony BDK Generator and follow instructions on screen `yo symphony`
+3. Link generator `npm link` _(required after development in the project)_
```

Comment:
is that something relevant for the users or for the developers only?

Reply:
Well it's more for developers to test the own changes. I remove it in next commit